pkgs/tools/networking/strongswan/default.nix

```
@@ -19,14 +19,16 @@ stdenv.mkDerivation rec {
   pname = "strongswan";
   version = "5.8.1"; # Make sure to also update <nixpkgs/nixos/modules/services/networking/strongswan-swanctl/swanctl-params.nix> when upgrading!
 
-  src = fetchurl {
-    url = "https://download.strongswan.org/${pname}-${version}.tar.bz2";
-    sha256 = "034rd6kr1bmnvj8rg2kcxdjb0cgj3dn9310mmm94j1awxan71byr";
+  src = fetchFromGitHub {
+    owner = "strongswan";
+    repo = "strongswan";
+    rev = "${version}";
+    sha256 = "1a1hw2jsbwvkdhhxjmq87hz13ivbgvqwks1q3adz14mqgbc64snd";
   };
 
   dontPatchELF = true;
 
-  nativeBuildInputs = [ pkg-config autoreconfHook ];
+  nativeBuildInputs = [ pkg-config autoreconfHook perl gperf yacc flex ];
```

```suggestion
  nativeBuildInputs = [ pkg-config autoreconfHook perl gperf bison flex ];
```
ofborg fails because yacc is a alias to bison and alises are not allowed in nixpkgs
```
  yacc = bison; # moved from top-level 2021-03-1
```